zeppelin-interpreter/src/main/java/org/apache/zeppelin/cluster/meta/ClusterMeta.java

```diff
@@ -51,12 +51,14 @@
   public static String MEMORY_CAPACITY      = "MEMORY_CAPACITY";
   public static String MEMORY_USED          = "MEMORY_USED";
 
-  public static String HEARTBEAT            = "HEARTBEAT";
+  public static String LAST_HEARTBEAT       = "LAST_HEARTBEAT";
```

Comment:
LATEST_HEARTBEAT ?

Reply:
I am spelling wrong, I am correcting it.

Reply:
I am spelling wrong, I am correcting it.